### PR TITLE
update

### PR DESCRIPTION
**Pull request by ghost**
fixed loading problems with xhr by changing the location of where the scripts loaded.

### CODE REVIEW

File: elements.js [hunk 1]
```diff
@@ -224,9 +224,10 @@ xtags["my-data-tbody"] = xtag.register("my-data-tbody",{
     extends:"tbody",
     lifecycle:{
         created: function () { 
-            this.init();
         },
-        inserted: function () { },
+        inserted: function () { 
+            this.init();
```
Comment (ghost) [inline elements.js:229]: This doesn't need to be moved.

**Comment by ghost** [comment]
Sorry about the sloppiness.  I just pushed the changes without cleaning up the code...

**Comment by kahshiu** [comment]
Looks like a whole bunch of test code in there, so I wont merge this edit
into my master. But yeah, I get your idea of settimeout use.

**Comment by ghost** [comment]
Yeah I know, sorry. I didn't know if the solution works for you in your build or if it would cause problems down the road so I left them as test functions.

**Comment by ghost** [comment]
I'm going to close this. 

Typically the method outlined in the PR isn't used for connecting to the `READY` callback, inline script elements cause problems for HTMLElements because of parsing of the inline script versus the parsing of the HTMLElement module and I stay away from it. 

If you want a PR request for x-tags still it would take a bit of work, I'd be happy to help but don't have time to get into the whole thing for you at the moment. 
Here is a link to a framework that uses an encapsulated shadow root host and also connects to the ready callback. 
https://github.com/doriansmiley/lotusJS-MWV

It uses x-tag and I've worked with it a bit but haven't really come up with any software solution for it.

**Comment by kahshiu** [comment]
Ok. Thanks for the response

**Pull request closed without merging**